twitter/View Controllers/TimelineViewController.m

```diff
@@ -48,4 +74,33 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 */
 
 
+- (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    TweetCell *cell = [tableView dequeueReusableCellWithIdentifier:@"TweetIdCell" forIndexPath:indexPath];
+    Tweet *tweet = self.arrayOfTweets[indexPath.row];
+    
+    NSString *URLString = tweet.user.profilePicture;
+    NSURL *url = [NSURL URLWithString:URLString];
+    NSData *urlData = [NSData dataWithContentsOfURL:url];
+    cell.profilePicture.image = [UIImage imageWithData:urlData];
```

Comment:
I actually really like that you decorated the cell from here instead of passing a tweet to a cell. My only nit would be that this is going to get messy so you can move this decoration to a helper method.